src/sgame/botlib/bot_nav_edit.cpp

```diff
@@ -439,15 +439,12 @@ void Cmd_NavTest( gentity_t *ent )
 	}
 }
 
-void NavEditInit()
-{
-	memset( &cmd, 0, sizeof( cmd ) );
```

Comment:
Is there an assurance somewhere that static-global variables get zero-initialised ? I know this is true for Linux elf and MacOS, but I'm not sure this is portable

Reply:
C++ standard